input-profile.js

```diff
@@ -0,0 +1,164 @@
+WL.registerComponent(
+	"input-profile",
+	{
+		controller: {type: WL.Type.Object},
+		handednessIndex: {
+			type: WL.Type.Enum,
+			values: ["left", "right"],
+			default: "left",
+		},
+		path: {
+			type: WL.Type.String,
+			default:
+				"https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@1.0/dist/profiles/",
```

Comment:
You might want to update this now that the Pico controllers are available? It's unfortunate that it doesn't auto-update... maybe there's a way to always use `latest` or something?

Reply:
unfortunatey there is no latest tag.
Although we can install them via 
```
npm install @webxr-input-profiles/assets
```
but it may increase dependancy size , 
or I can just manually replace the url with latest one ( but it requires manual updation now and then)